specs.json

```diff
@@ -1276,6 +1278,12 @@
   "https://www.w3.org/TR/html-aam-1.0/",
   "https://www.w3.org/TR/html-aria/",
   "https://www.w3.org/TR/html-media-capture/",
+  {
+    "url": "https://www.w3.org/TR/html-ruby-extensions/",
```

Comment:
hmm... This is in practice a delta spec of the HTML spec (although not intended to serve as a next iteration of it); should this be represented somehow?

In particular, it adds 2 HTML elements that I suspect we want to detect.

Reply:
> This is in practice a delta spec of the HTML spec (although not intended to serve as a next iteration of it); should this be represented somehow?

I was thinking we would handle that as we handle TC39 proposals: at some point in the future, we flag the entry as discontinued and obsoleted by HTML. It's not easy to flag the spec as being a "delta" of "HTML" (code would get confused by a `seriesPrevious` property in `specs.json` in particular)

> In particular, it adds 2 HTML elements that I suspect we want to detect.

They should end up being detected. But the ones that already exist in HTML will be detected as well. We're going to end up with duplicates in extracts, including duplicates of exported definitions such as the `ruby` element, which may be more problematic for Bikeshed. Nothing that should be blocking though?

Reply:
not blocking for sure; the reason I'm mentioning it is in case we had an annotation here that would help with that conflict down the line.

Reply:
Tracked in #1488. In the meantime, I'll warm up for a little Webref patching exercise.